boards/nrf51dk/include/periph_conf.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf51dk
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the Nordic nRF51DK
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "periph_conf_common.h"
+#include "cfg_clock_16_1.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name UART configuration
+ * @{
+ */
+#define UART_NUMOF          (1U)
+/* UART pin configuration */
```

Comment:
is this board really using UART hw flow control? Just asking, as the `nrf52xxxdk´ boards are IMO not doing so. Or does the uart-to-serial (jlink) chip on those boards allow for both?!

Reply:
From the User guide (see section 4.2 Virtual COM Port), there is hw flow control on the interface MCU. It's also possible to set a baudrate up to 1Mbps but I haven't tried.
Maybe this could also explained why I couldn't have a fully working UART using the JLink firmware. I'll give another try tomorrow.